mlrun/datastore/sources.py

```diff
@@ -31,6 +30,9 @@
 import mlrun
 from mlrun.config import config
 from mlrun.datastore.snowflake_utils import get_snowflake_spark_options
+from mlrun.datastore.utils import (
+    transform_list_filters_to_tuple,
+)
```

Comment:
Ruff seems to be lazy, not changing this either way, so I'll take it up on myself to comment :)
```suggestion
from mlrun.datastore.utils import transform_list_filters_to_tuple
```